app/models/device_story_comment.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DeviceStoryComment < ApplicationRecord
+  include Rakismet::Model
+  validates :content, presence: true, length: { maximum: 1000 }
+  validates :user_id, presence: true
+  validates :device_story_id, presence: true
+  belongs_to :device_story
+  belongs_to :user
```

Comment:
It might be better to use `require` options instead of validating ids explicitly.

```suggestion
  belongs_to :device_story, require: true
  belongs_to :user, require: true
```